ci_framework/roles/openshift_login/tasks/login.yml

```diff
@@ -14,6 +14,31 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Fetch user password from file in case is not provided by a variable
+  when:
+    - cifmw_openshift_login_password is not defined
```

Comment:
Hmm... I just spent few minutes trying to understand how it is working, when I finally noticed the difference between
`cifmw_openshift_login_password != cifmw_openshift_login_password_file` and 
`cifmw_openshift_password != cifmw_openshift_password_file`

Reply:
I'm not sure how to make it more readable :(